src/medium-zoom.js

```diff
@@ -18,24 +18,32 @@ const isNode = selector => selector && selector.nodeType === 1
  *
  * @param {(string|Element[])} selector The selector to target the images to attach the zoom to
  * @param {object} options The options of the zoom
- * @param {number} [options.margin=0] The space outside the zoomed image
```

Comment:
We should actually keep this option. The margin here is not a style applied to the container or the image. It is a virtual margin, a value that is used in position calculations everywhere (for animations etc.).